fixed sidebar button first click not working bug, added image element…

haydenkarl22 opened the pull request:
- Fixed bug where when html file first loaded, the sidebar button would not properly load the sidebar on the first click and first click only.
- Removed home from footer
- added img elements to each footer button (images don't load currently, I need tech leads to send me the pictures they desire for these buttons from rerum.io(rerum.io has similar footer, that repo has the pictures I need), once I get the desired images I can get them saved in the /images folder in our repo and use those.)
- subtle footer styling changes: the goal with these subtle changes was to make the footer buttons fit exactly in the footer, as well as changing the alignment from left alignment to center alignment.

sesheppard commented on web/tools.html:
```diff
@@ -156,21 +167,21 @@ <h4>Tools</h4>
-                    Home
-                </a>
                 <a href="https://rerum.io/#/annotation" class="button">
+                    <img src="/web/images/rerum_logo.png" alt="Annotation Icon">
```
These icons appear to have a special class in a style sheet on the main RERUM page. Is there a way we can find the original source of the images from that, or load the same styles ourselves?

alar12 replied:
You can find the images in this repository [RERUM](https://github.com/CenterForDigitalHumanities/rerum)

sesheppard commented on web/tools.html:
```diff
@@ -181,7 +192,7 @@ <h4>Tools</h4>
             function openCloseMenu() {
                 var toolBar = document.getElementById("toolBar");
-                if (toolBar.style.width === "0px") {
+                if (toolBar.style.width === "0px" || toolBar.style.width === "") {
```
I have seen the page content that is to be shifted aside when the menu bar is opened wrapped into a class of its own before in order to avoid the overlap, so the main content width can be changed with an alteration to the style similar to this. Is there a different way to achieve the same effect that we could implement?

kate-holdener commented:
> * Fixed bug where when html file first loaded, the sidebar button would not properly load the sidebar on the first click and first click only.
> * Removed home from footer
> * added img elements to each footer button (images don't load currently, I need tech leads to send me the pictures they desire for these buttons from rerum.io(rerum.io has similar footer, that repo has the pictures I need), once I get the desired images I can get them saved in the /images folder in our repo and use those.)
> * subtle footer styling changes: the goal with these subtle changes was to make the footer buttons fit exactly in the footer, as well as changing the alignment from left alignment to center alignment.

Please use:
Fixes #13
In the PR description

alar12 commented on web/tools.html:
```diff
@@ -83,35 +83,27 @@
                 height: 38px;
-                padding: 0 30px;
+                padding: 0 10px;
                 color: #555;
```
Create a new file for styling we should not be using internal CSS

alar12 commented on web/tools.html:
```diff
@@ -156,21 +167,21 @@ <h4>Tools</h4>
 
         <footer>
-            <!--Buttons from RERUM Home Page. Not properly reflexive currently, and missing the correct font.-->
             <div class="footerButtons">
```
use separate html file for the footer(footer.html) as of now the changes are applied only to the tools.html.

alar12 reviewed (dismissed):
Good Work!
Create a new file for styling we should not be using internal CSS. Use separate html file for the footer(footer.html) as of now the changes are applied only to the tools.html(Menu Toggle Issue too) and does not effect all the other pages. And the images of footer section are in this repository [RERUM](https://github.com/CenterForDigitalHumanities/rerum). Also the Menu Bar Responsiveness is sill the same - Make the menu responsive so it does not obstruct homepage content. Lastly update the menu bar text option(home, about, contribute, etc..) colors for better visibility.

haydenkarl22 commented:
I've made the all but one of the desired changes suggested by @alar12 

1. Removed footer element from tools.html and created a footer.html file that contains the footer content. 
2. developed fetch call that fetches footer for the tools.html page.
3. separated stylings to CSS file labeled styles.css (Both this file and footer.html are in the same /web folder as tools.html)
4. removed jittery sidebar bug where sidebar covers material in the body of the page.

The only change I have not made is updated the footer icons to be the same as the webpages in the RERUM repo. I attempted to locate these images by scanning through the repo, and using inspect element on the rerum.io webpage, but I could not find these files, so if someone could direct me to their location in the RERUM repo, I can update these footer images in no time. 

These changes can be viewed by running this command in terminal: python -m http.server 8000 in the /web folder, then opening http://localhost:8000/tools.html in any browser. 

Here is how the page looked after sprint progress last week:
![image](https://github.com/user-attachments/assets/04fe72c0-25c7-45fb-9e32-3f82db11aae9)

And here is the current look after my push:
![image](https://github.com/user-attachments/assets/c778551e-93da-4337-a874-afbc34592913)

Also, Here is proof the sidebar no longer covers the body:
![image](https://github.com/user-attachments/assets/a149617c-34c4-4953-8f60-546a26012da4)
